packages/legend-graph/src/graphManager/protocol/pure/v1/engine/V1_EngineServerClient.ts

```diff
@@ -603,8 +604,8 @@ export class V1_EngineServerClient extends AbstractServerClient {
       {},
       undefined,
       serviceExecutionMode === ServiceExecutionMode.FULL_INTERACTIVE
-        ? { storeModel: TEMPORARY__useStoreModel }
-        : undefined,
+        ? { storeModel: TEMPORARY__useStoreModel, generateLineage: TEMPORARY__useGenerateLineage }
+        : {undefined, generateLineage: TEMPORARY__useGenerateLineage},
```

Comment:
should it be {storeModel: undefined, generateLineage: TEMPORARY__useGenerateLineage}?

Reply:
I am adding generateLineage as parts of the Service Registration options, so it should be inside the same curly brackets as store model right? I thought so because this { } holds the argument for all options, but pls let me know if I have the idea wrong